specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -6003,6 +6204,395 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Time in UTC at which the message was created.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "LabelingJob": {
+      "description": "Definition of a labeling job.",
+      "type": "object",
+      "properties": {
+        "labelCategories": {
+          "description": "Label categories of the job.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/LabelCategory"
+          }
+        },
+        "jobInstructions": {
+          "description": "Instructions for the job.",
+          "$ref": "#/definitions/LabelingJobInstructions"
+        },
+        "datasetConfiguration": {
+          "description": "Dataset configuration for the job.",
+          "$ref": "#/definitions/LabelingDatasetConfiguration"
+        },
+        "mlAssistConfiguration": {
+          "description": "Machine learning assisted configuration for the job.",
+          "$ref": "#/definitions/MLAssistConfiguration"
+        },
+        "labelingJobMediaProperties": {
+          "description": "Media specific properties in a labeling job.",
+          "$ref": "#/definitions/LabelingJobMediaProperties"
+        },
+        "projectId": {
+          "format": "uuid",
+          "description": "Internal id of the job(Previously called project).",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the job.",
+          "$ref": "#/definitions/JobStatus",
+          "readOnly": true
+        },
+        "progressMetrics": {
+          "description": "Progress metrics of the job.",
+          "$ref": "#/definitions/ProgressMetrics",
+          "readOnly": true
+        },
+        "statusMessages": {
+          "description": "Status messages of the job.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusMessage"
+          },
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Created time of the job in UTC timezone.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "Name and identifier for a job.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The job property dictionary. Properties can be added, but not removed or altered.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tags": {
+          "description": "The job tag dictionary. Tags can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
```

Comment:
Usually in a resource we put the `tags` in the root level of the model, rather than in the `properties` field. Is it possible to move this to `LabelingJobResource`?

Reply:
In the second iteration, I followed the pattern Jennny did in her PR: https://github.com/Azure/azure-rest-api-specs/pull/10421 for consistency between job models in MachineLearningServices, which has `Tags` in the inner payload other than the resource level.  Also, `Tags` and `Properties` are 2 dictionaries every resource supply. Since the `LabelingJobResource` already has a field named "Properties`, which is the "LabelingJobProperties" I feel it will be less confusing to keep these 2 dictionaries under the inner level.